src/Mutator/NodeMutationGenerator.php

```diff
@@ -82,7 +82,7 @@ public function __construct(
     }
 
     /**
-     * @return Mutation[]
+     * @return \Infection\Mutation\Mutation[]
```

Comment:
why is it needed? I think should be reverted

Reply:
Happens occasionally when doing a refactor to change the class namespace with phpStorm